models/business-unit/src/company/company-draft/generator.ts

```diff
@@ -0,0 +1,28 @@
+import { AddressDraft } from '@commercetools-test-data/commons';
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import { status, storeMode, unitType, associateMode } from '../../constants';
+import type { TBusinessUnitDraft } from '../../types';
+
+// https://docs.commercetools.com/api/projects/business-units#businessunitdraft
```

Comment:
Should this be 'https://docs.commercetools.com/api/projects/business-units#companydraft'?